src/box/lua/iproto.lua

```diff
@@ -0,0 +1,28 @@
+-- Sets IPROTO request handler callback (second argument) for the given request
+-- type (first argument, number).
+-- Passing nil as the callback resets the corresponding request handler.
+box.iproto.override = function(request_type, callback)
+    local trigger = require('trigger')
+
+    if request_type == nil then
+        box.error(box.error.PROC_LUA,
+                  'Usage: box.iproto.override(request_type, callback)')
+    end
+    if type(request_type) ~= 'number' then
+        box.error(box.error.PROC_LUA, 'expected uint64_t as 1 argument')
+    end
+    if callback ~= nil and type(callback) ~= 'function' then
+        box.error(box.error.PROC_LUA,
+                  ("bad argument #2 to 'override' (function expected, " ..
+                   "got %s)"):format(type(callback)))
+    end
+
+    local event_name = ('box.iproto.override[%d]'):format(request_type)
+    local trigger_name = '__tarantool__internal__iproto__override__trigger__'
```

Comment:
I think it's OK to set a handler with `box.iproto.override` and then delete it with trigger registry.
So let's call a trigger with more readable name (`box.iproto.override.lua.user_handler`, for example).

Reply:
Done.